modules/emulator/R/minimize.GP.R

```diff
@@ -97,6 +97,9 @@ gpeval <- function(xnew, k, mu, tau, psi, x, rng, splinefcns) {
 ##' @name ddist
 ##' @title ddist
 ##' @export
+##' 
+##' @param x Name of variable to plot on X axis
+##' @param prior 'unif', 'IG'
```

Comment:
data.frame specifying a prior probability distribution in terms of the distribution name (distn) and first and second parameters (parama, paramb)